packages/project/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export * as project from "./project";
```

Comment:
and in addition I would move these to src/db folder since these are all db specific functions